app/ante/msg_gatekeeper.go

```diff
@@ -14,7 +14,7 @@ var (
 	_ baseapp.CircuitBreaker = MsgVersioningGateKeeper{}
 )
 
-// MsgVersioningGateKeeper dictates which transactions are accepted for an app version
+// MsgVersioningGateKeeper dictates which transactions are accepted for an the app version
```

Comment:
let's revert this b/c it doesn't make sense

```suggestion
// MsgVersioningGateKeeper dictates which transactions are accepted for an app version
```